app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">ParkingSystemJava</string>
-</resources>
\ No newline at end of file
+    <string name="configure_parking_size">configure parking size</string>
+    <string name="welcome">Welcome!</string>
```

Comment:
please use a more descriptive name and follow the naming conventions suggested
https://jeroenmols.com/blog/2016/03/07/resourcenaming/

Reply:
done